test/Error/error_cause.js

```diff
@@ -0,0 +1,64 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Copyright (c) 2021 ChakraCore Project Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+
+
+const o = {};
+const tests = [
+    {
+        name: "Error.prototype.cause",
+        body: function () {
+            assert.isFalse("cause" in Error.prototype, "Cause property must not exist in *Error.prototype");
+        }
+    },
+    {
+        name: "Error().cause === undefined",
+        body: function () {
+            assert.isFalse("cause" in SyntaxError(), "Error().cause should not be defined if specified by cause property of options parameter in *Error constructor");

```

Comment:
I don't understand the error message here - do you mean "Error().cause should not be defined if specified by cause property of options parameter in *Error constructor"